Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectGenerator/MSBuildTemplates/SDKProjectTemplate.csproj.template

```diff
@@ -75,6 +75,9 @@
   <!--IMPORT SDK.props manually to be able to set the BaseIntermediateOutputPath above path-->
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
   
+  <PropertyGroup>
```

Comment:
As discussed offline. Approving the change, but it does feel a bit wrong to me to be making a change to support building on older versions of DotNet SDK. It seems like we should instead look into the [global.json file](https://docs.microsoft.com/en-us/dotnet/core/versions/selection#the-sdk-uses-the-latest-installed-version) as a way to specify the minimum dotnet SDK.

Reply:
Yep, we can handle this in the future.